webpack.config.js

```diff
@@ -1,24 +1,51 @@
 const path = require('path');
```

Comment:
Since we are compiling the bundle on dev for now and commiting the current bundle into the repository, we definitely need a piece on the readme about how to compile